### DISCUSSION TITLE
123

### DISCUSSION THREAD

**Discussion by ghost** [azerothcore/azerothcore-wotlk#13765] [Job offers]

**Comment by virgo77** [comment]
Hi, did you follow this [wiki](https://www.azerothcore.org/wiki/installation)?
If yes, you should have a running fresh server from scratch

More informations about your server and how you installed it would be appreciated to give you better answers, because you say that you "recently got a server installed [wowltk]..." (it is WotLK, huh).
So where is your server from (AzerothCore, TrinityCore, MaNGOS / cMaNGOS / EasyMaNGOS)?

Thanks to give us all answers.

See you.

V.

**Comment by virgo77** [comment]
Please, can a moderator close this topic?
the first topic has been totally cleaned.
Thanks
